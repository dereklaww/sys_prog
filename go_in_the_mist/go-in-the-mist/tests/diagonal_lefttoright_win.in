who
place A1
place B1
place B2
place C1
place C3
place D1
who
view
place D4
place E1
place E5