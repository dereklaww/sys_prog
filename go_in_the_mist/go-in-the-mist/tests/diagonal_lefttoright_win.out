B
B
B3,xx.....xx.....xx.....xx..#..xx.#...xx#ooo.xxxxxxx
Black wins!
A1B1B2C1C3D1D4E1E5
Thank you for playing!